ssl_cert_tracker/utils.py

```diff
@@ -14,19 +14,20 @@
 
 """
 import logging
-from datetime import datetime
 import xml.dom.minidom
+from dateutil import parser
 
 logging.basicConfig(filename='p_soc_auto.log', level=logging.DEBUG)
 
 def validate(date_text):
     """check if date_text is a valid date  """
     try:
-        if date_text != datetime.strptime(date_text, "%Y-%m-%d").strftime('%Y-%m-%d'):
-            raise ValueError
+        parser.parse(date_text[0:10])

```

Comment:
why [0:10]?

Reply:
the first 10 character represents date, I have not seen any thing otherwise, for for faster processing it is better to check the first 10 characters
You may say what if they change the format, then we will address it again (of course they could change representation of any other field as well)